### ISSUE TITLE
Windows version fails to download youtube-dl executable.

### ISSUE THREAD

**Issue by Open96** [Open96/JYpm#1]
Linux version works fine. Will fix soon.

**Comment by 912d** [comment]
I had similar situation and resolved it, temporarily, by commenting out this if statement
https://github.com/Open96/JYpm/blob/release/jypm-internet-checker/src/main/java/com/github/open96/jypm/internetconnection/ConnectionChecker.java#L83

Later, after work, I'll dig deeper into this topic

**Comment by Open96** [comment]
Let me know if you find anything. I have not had much time recently to fix that, but eventually I plan to make a fix as soon as I have some free time.

**Comment by Open96** [comment]
Alright, when running ping command via cmd this is what I get:

```
C:\Users\user1>ping -c 1 google.com

Access denied. Option -c requires administrative privileges.
```

So pretty much what needs to be done is to have a special command for Windows that doesn't need administrative privileges. 

It worked quite well so far on Windows so I guess either Microsoft blocked it in some patch or my method never worked as it started to break after patch that fixed YoutubeDlManager downloading new executable on every application startup regardless of ConnectionChecker's output.

**Comment by Open96** [comment]
Fixed issue in commit 5e1a6aedd58098c6dbeb06be58d412e6bbda9a9f.
